src/ServiceFabric/ServiceFabric/Template/Windows/template.json

```diff
@@ -829,7 +834,7 @@
           }
         ],
         "provisioningState": "Default",
-        "reliabilityLevel": "[parameters('reliabilityLevel')]",
+        "reliabilityLevel": "Silver",
```

Comment:
same thing here wonder if we should just validate you cant pass bronze but allow for gold etc

Reply:
updated